src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/arithmetic.txt

```diff
@@ -30,6 +30,9 @@
 >> 5-3
 2
 
+>> 5+-3
+2
+
```

Comment:
A test validating precedence would also be good, something like 
1 - 2 * 3

Reply:
Extra tests have been added.